### PR TITLE
Drop DqlSelection and Dql namespace

### PR DESCRIPTION
**Pull request by foxycode**
Related to #238

### CODE REVIEW

**Comment by foxycode** [comment]
@fprochazka This one should be finished too.

**Comment by foxycode** [comment]
@fprochazka Rebased from master

File: src/Kdyby/Doctrine/Dql/InlineParamsBuilder.php [hunk 1]
```diff
@@ -1,107 +0,0 @@
...
-use Kdyby;
-use Kdyby\Doctrine\Helpers;
-use Nette;
-
-
-
-/**
- * @author Filip Procházka <filip@prochazka.su>
- */
-class InlineParamsBuilder extends Kdyby\Doctrine\QueryBuilder
```
Comment (fprochazka) [inline src/Kdyby/Doctrine/Dql/InlineParamsBuilder.php]: do not remove this one please

File: src/Kdyby/Doctrine/EntityRepository.php [hunk 2]
```diff
@@ -187,7 +187,7 @@ public function findAssoc($criteria, $key = NULL)
 	 *
 	 * @param string|NULL $alias
 	 * @param string|NULL $indexBy
-	 * @return \Kdyby\Doctrine\DqlSelection
+	 * @return \Kdyby\Doctrine\QueryBuilder
 	 */
 	public function select($alias = NULL, $indexBy = NULL)
```
Comment (fprochazka) [inline src/Kdyby/Doctrine/EntityRepository.php]: this method should go too

File: tests/KdybyTests/Doctrine/InlineParamsQueryBuilder.phpt [hunk 3]
```diff
@@ -1,216 +0,0 @@
...
-
-require_once __DIR__ . '/../bootstrap.php';
-require_once __DIR__ . '/models/cms.php';
-
-
-
-/**
- * @author Filip Procházka <filip@prochazka.su>
- */
-class InlineParamsQueryBuilderTest extends KdybyTests\Doctrine\ORMTestCase
```
Comment (fprochazka) [inline tests/KdybyTests/Doctrine/InlineParamsQueryBuilder.phpt]: also this one should stay too

File: src/Kdyby/Doctrine/Dql/InlineParamsBuilder.php (hunk 1 above)
Comment (foxycode) [inline src/Kdyby/Doctrine/Dql/InlineParamsBuilder.php]: @fprochazka You wanted to remove whole Dql namespace. Where should I move it then?

Comment (fprochazka) [inline src/Kdyby/Doctrine/Dql/InlineParamsBuilder.php]: Yeah I know, but I forgot about this one. Just leave it for now please.

**Comment by foxycode** [comment]
@fprochazka Updated as you wanted. I renamed `InlineParamsBuilder` to `InlineParamsQueryBuilder` to reflect test name, which seems more proper than renaming test. Hope it's ok.

**Comment by fprochazka** [comment]
@foxycode well, how many people are extending tests and how many people are extending classes that are part of a library? I don't think renaming the class is a good idea at this point.

**Comment by fprochazka** [comment]
I do agree that is probably a better name, but I don't wanna think about it now and introduce more BC Breaks than neccesary.

**Comment by foxycode** [comment]
@fprochazka Ok, should I rename test then or leave it as is?

**Comment by fprochazka** [comment]
Also, by renaming the class, you're combining two unrelated changes in one PR.

Please revert the rename.

**Comment by fprochazka** [comment]
Please revert the rename. It is not neccesary right now nor related.

**Comment by foxycode** [comment]
@fprochazka Done

**Comment by fprochazka** [comment]
@foxycode thank you! :)

**Comment by fprochazka** [comment]
Again: if anybody was using it, you're welcome to create a new package with this (or better) code that implements it.

**Pull request merged**